source/_integrations/binary_sensor.modbus.markdown

```diff
@@ -27,6 +27,7 @@ binary_sensor:
         hub: hub1
         slave: 1
         coil: 110
+        register_type: discrete_input
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Thanks for the clarification, offending line has been removed